ddtrace/tracer/span.go

```diff
@@ -580,4 +580,14 @@ const (
 	keyTopLevel = "_dd.top_level"
 	// keyPropagationError holds any error from propagated trace tags (if any)
 	keyPropagationError = "_dd.propagation_error"
+	// keySpanSamplingMechanism specifies the sampling mechanism by which an individual span was sampled
+	keySpanSamplingMechanism = "_dd.span_sampling.mechanism"
+	// singleSpanSamplingMechanism specifies value reserved to indicate that a span was kept
+	// on account of a single span sampling rule.
+	samplingMechanismSingleSpan = 8
```

Comment:
This feels out of place here 😄 Can we maybe make a different `const (...)` group?